packages/core-js/internals/object-create.js

```diff
@@ -45,6 +67,6 @@ module.exports = Object.create || function create(O, Properties) {
     EmptyConstructor[PROTOTYPE] = null;
     // add "__proto__" for Object.getPrototypeOf polyfill
     result[IE_PROTO] = O;
-  } else result = NullProtoObject();
+  } else result = viaActiveX() ? NullProtoObjectViaActiveX() : NullProtoObjectViaIFrame();
```

Comment:
It's the creation of 2 (!!!) ActiveX objects on each `Object.create(null)`. Also, `Object.prototype` from ActiveX is not cleared from properties.

Reply:
👌 check the latest commit